src/content/posts/2017-08-07-dddsydney-2017-recap.md

```diff
@@ -32,12 +32,12 @@ While attendee growth is important we also saw a growth in speakers, we had arou
 
 ## Listening to Feedback
 
-While there was a bunch of stuff to address just based on what we learnt doing it the first time we did listen to the feedback that our attendees provides us with. One of the main pieces of feedback from the 2016 event was around the food, we only _just_ had enough food last year. This year I decided to listen to my wifes advice and let her take the lead on that, after all she has worked in marketing/sales in the food industry for the last 7-odd years. The main thing she pushed for from our supplier was pre-packed lunchboxes for lunch rather than the traditional sandwich platter/hot trays/etc. and this worked a treat. By having everything pre-packaged lunch was a 'pick a box and go' for attendees and it removed the feeling of inequality ("hey, they took a bigger plate than me!"). Having read this years feedback it the food was a highlight (well, behind content and people obviously :winking:), the only negative food feedback was that we had too much!
+While there was a bunch of stuff to address just based on what we learnt doing it the first time we did listen to the feedback that our attendees provides us with. One of the main pieces of feedback from the 2016 event was around the food, we only _just_ had enough food last year. This year I decided to listen to my wifes advice and let her take the lead on that, after all she has worked in marketing/sales in the food industry for the last 7-odd years. The main thing she pushed for from our supplier was pre-packed lunchboxes for lunch rather than the traditional sandwich platter/hot trays/etc. and this worked a treat. By having everything pre-packaged lunch was a 'pick a box and go' for attendees and it removed the feeling of inequality ("hey, they took a bigger plate than me!"). Having read this years feedback it seems the food was a highlight (well, behind content and people obviously :winking:), the only negative food feedback was that we had too much!
```

Comment:
I'm not too sure this is what you meant but it read funny without a verb in here. If it's not, trash the change.